src/main/java/chalmers/pimp/model/Stroke.java

```diff
@@ -17,16 +17,18 @@
   private final List<IPixel> pixels;
   private final ModelMemento modelMemento;
   private final int diameter;
+  private final IColor color;
 
   /**
    * @param modelMemento the model memento object, that represents the current model state.
    * @param diameter     the diameter of the stroke, in pixels.
    * @throws NullPointerException if any arguments are {@code null}.
    */
-  Stroke(ModelMemento modelMemento, int diameter) {
+  Stroke(ModelMemento modelMemento, int diameter, IColor color) {
     this.modelMemento = Objects.requireNonNull(modelMemento);
     this.diameter = diameter;
     pixels = new ArrayList<>(10);
+    this.color = color;
   }
```

Comment:
Missing @param tag for color. Also the ctor doesn't throw a NullPointerException if the color is null.